platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/HTMLParser.swift

```diff
@@ -133,9 +133,9 @@ public final class HTMLParser {
         if mutableAttributedString.string.last == "\n",
            !html.hasSuffix("</code>"),
            !html.hasSuffix("</a>"),
-           !html.hasSuffix("</p><p></p></blockquote>"),
-           !html.hasSuffix("</ul><p></p></blockquote>"),
-           !html.hasSuffix("</ol><p></p></blockquote>"),
+           !html.hasSuffix("</p><p>\(Character.nbsp)</p></blockquote>"),
+           !html.hasSuffix("</ul><p>\(Character.nbsp)</p></blockquote>"),
+           !html.hasSuffix("</ol><p>\(Character.nbsp)</p></blockquote>"),
```

Comment:
@aringenbach this fixed a bug where newlines where not added anymore to blockquotes however... I think that actually the nbsp character should be removed from here, because we are now not able anymore to add more than one consecutive newline inside a blockquote